starport/services/network/networktypes/genesisinformation.go

```diff
@@ -0,0 +1,79 @@
+package networktypes
```

Comment:
Can we move this to the network pkg?

Reply:
Why? The information is used by `networkchain` so we don't have a `network` dependency in `networkchain`

Reply:
ok I just noticed that. but still, I'm a bit un comfortable with with the networktypes pkg in general. because it feels like we define all of the types of network pkg to there whereas we only put the shared types there between the network and networkchain.

Anyways, I'll try to make a refactoring on this later and see if we can rm this pkg completely.

Reply:
> we define all of the types of network pkg to there whereas we only put the shared types there between the network and networkchain.

I didn't get this statement. The point is actually to define what is shared between `network` and `networkchain`. Example, `networkchain` should be agnostic to `spn` communication but it should still have somewhere the concept of a genesis account in order to build the chain with Genesis Information

I'm not closed to a refactoring for this but I think it's better if we discuss the new architecture before creating a PR